recipes/powerci.rb

```diff
@@ -88,3 +88,112 @@
 )
 
 include_recipe 'osl-jenkins::master'
+
+# Jenkins needs to be restarted before running the following script otherwise
+# the imports fail
+jenkins_command 'safe-restart'
```

Comment:
This is the output from a converge without restarting Jenkins.

```
STDOUT: 
       STDERR: ERROR: Unexpected exception occurred while performing groovy command.
       org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
       RemoteClass: 7: unable to resolve class com.nirima.jenkins.plugins.docker.DockerTemplate
        @ line 7, column 1.
          import com.nirima.jenkins.plugins.docker.DockerTemplate
          ^
       
       RemoteClass: 9: unable to resolve class com.nirima.jenkins.plugins.docker.launcher.DockerComputerSSHLauncher
        @ line 9, column 1.
          import com.nirima.jenkins.plugins.docker.launcher.DockerComputerSSHLauncher
          ^
       
       RemoteClass: 10: unable to resolve class hudson.plugins.sshslaves.SSHConnector
        @ line 10, column 1.
          import hudson.plugins.sshslaves.SSHConnector
          ^
       
       RemoteClass: 6: unable to resolve class com.nirima.jenkins.plugins.docker.DockerCloud
        @ line 6, column 1.
          import com.nirima.jenkins.plugins.docker.DockerCloud
          ^
       
       RemoteClass: 8: unable to resolve class com.nirima.jenkins.plugins.docker.DockerTemplateBase
        @ line 8, column 1.
          import com.nirima.jenkins.plugins.docker.DockerTemplateBase
          ^
       
       5 errors
       
```
Maybe I just need to add those plugins to `['osl-jenkins']['restart_plugins']` ?